### PR TITLE
Rename Scans to DAST Scanner

### PR DESCRIPTION
**Pull request by mahmoudadly-cobalt**
### Updated
Scans -> DAST Scanner
https://deploy-preview-468--cobalt-docs.netlify.app/platform-deep-dive/scans/

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Scans/_index.md [hunk 1]
```diff
@@ -12,16 +12,17 @@ Scan your Web Applications to discover external vulnerabilities.
...
-To access the Scans Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
+To access the DAST Scanner Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
 
 We use the following IP Address to make requests to your target: **52.19.40.38**
 
-## How to Use Scans
+## How to Use DAST Scanner
 
+<!-- TODO: update this video -->
 [Watch the how-to video here.](https://videos.cobalt.io/watch/4D8kQcTmPBE3HdWy6X3fHW)
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:26]: I noticed that the demo video we have will also need to be updated to show the recent UX.

Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:26]: I noticed that the demo video we have will also need to be updated to show the recent UX. @eugenerevzin  @mtsuji-cobalt

Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md:26]: It looks like this is on the To Do List for GA launch: 
Screenshot 2024-02-14 at 06 19 32

https://zombie.atlassian.net/wiki/spaces/MO/pages/2795962414/DAST+GA+Launch+Plan

File: content/en/Platform Deep Dive/Scans/_index.md [hunk 2]
```diff
@@ -12,16 +12,17 @@ Scan your Web Applications to discover external vulnerabilities.
 
 The complexity of today's applications, combined with the constant evolution of cyber threats, makes it increasingly difficult to ensure security measures. Traditional methods of security assessment often fall short of providing continuous insights, leading to potential blind spots.
 
-Scans address these challenges head-on. By combining the power of PtaaS with DAST, you gain a comprehensive solution for continuously assessing risk.
+DAST Scanner addresses these challenges head-on. By combining the power of PtaaS with DAST, you gain a comprehensive solution for continuously assessing risk.
```
[the remark this answers is not in the document]
Comment (mahmoudadly-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md]: I updated this line. Line 19 already updated. https://deploy-preview-468--cobalt-docs.netlify.app/platform-deep-dive/scans/

File: content/en/Platform Deep Dive/Scans/_index.md [hunk 3]
```diff
@@ -12,16 +12,17 @@ Scan your Web Applications to discover external vulnerabilities.
...
 
 The beta period runs until December 31, 2023. During the beta, you have access to three scans a month, allowing you to fully explore this new feature. Scans reset on the first of the month. Anticipate usage and limitation adjustments in 2024.
 
-To access the Scans Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
+To access the DAST Scanner Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
 
 We use the following IP Address to make requests to your target: **52.19.40.38**
 
-## How to Use Scans
+## How to Use DAST Scanner
```
Comment (mtsuji-cobalt) [inline content/en/Platform Deep Dive/Scans/_index.md:23]: missed this one, to add 'the' to: How to Use the DAST Scanner

Comment (mahmoudadly-cobalt) [reply content/en/Platform Deep Dive/Scans/_index.md]: AHa! The title too. Title grammar confuses me. 🤦‍♂️ 
Updated.

**Pull request merged**